directdemod/merger.py

```diff
@@ -41,7 +41,7 @@ def __init__(self, tle_file='', aux_file_name="temp_image_file.png"):
 
         Checker.check_libs()
 
-        self.is_basemap = Checker.check_basemap()
+        self.is_basemap = False
```

Comment:
Are we using only cartopy now?

Reply:
For georeferencing and overlay I use `gdal`, prototype of image merger uses `cartopy`, but when I rewrite it, it will use only `gdal` too.